### PR TITLE
Deletes reddit

### PR DESCRIPTION
**Pull request by GuyonBroadway**
Reddit is now removed from the premises.

:cl:Guyonbroadway
del: Deletes reddit
fix: fixed an issue with redditors on the server.
/:cl:

Chemotherapy is painful but its the only cure to reddit cancer.

### CODE REVIEW

**Comment by Tumesuo** [comment]
Much needed change.

**Comment by KomradeSpectre** [comment]
The scope of this isn't even good, you only made it affect human subtypes.

**Comment by GuyonBroadway** [comment]
Well spotted, thanks.

**Comment by DaedalusGame** [comment]
should ban ghosts if they say it

**Comment by AdmiralPancakes1** [comment]
hopline suicide buff

**Comment by mkikt4743** [comment]
good idea

**Comment by Punkinguy** [comment]
step 1: ban oranges
step 2: glass new zealand to prevent him from making alt accounts

**Comment by 81Denton** [comment]
phew, once this is merged we can finally pay attention to important features like #48057 The fortnite team member down sound effect now plays when getting sent to crit

**Comment by Michael-Ashfield** [comment]
Does this affect wicker doll forcing players to say things, or forced speech nanites gibbing 50% of the station?

**Comment by DaedalusGame** [comment]
i've changed my mind this is a webedit pr close he posthaste

**Comment by GuyonBroadway** [comment]
It's been tested locally.

**Comment by GuyonBroadway** [comment]
image

It was quicker to webedit the some 12 character change than it was to muck around with git.

**Comment by Ziiro** [comment]
please test merge this, I have a plan

File: code/modules/mob/living/say.dm [hunk 1]
```diff
@@ -116,6 +116,11 @@ GLOBAL_LIST_INIT(department_radio_keys, list(
 	if(findtext(message, " ", 1, 2))
 		message = copytext(message, 2)
 
+	if(findtext(message, "reddit"))
+		var/mob/living/H = src
```
Comment (JJRcop) [inline code/modules/mob/living/say.dm]: `src` is already mob/living

**Comment by trollbreeder** [comment]
haha funny
On a more serious note, maybe use the filter that's being used to prevent characters from saying "WTF" or "LMAO" instead of checking inside say and gibbing? Because gibbing is a bit harsh and makes this PR feel like it belongs in the april fools testmerge bin and because it's a lot more efficient to use systems already in place. I give this 0 reddit silvers out of 10

**Comment by optimumtact** [comment]
this is already filtered by the ic filter

**Comment by trollbreeder** [comment]
thanks oranges you take out the trash for us

**Pull request closed without merging**